documentation/library_guide/kernel_templates/esimd_main.rst

```diff
@@ -0,0 +1,17 @@
+ESIMD-Based Kernel Templates
+############################
+
+The ESIMD kernel templates are based on `Explicit SIMD SYCL extension <https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2024-0/explicit-simd-sycl-extension.html>`_ of Intel® oneAPI DPC++/C++ Compiler.
+This technology only supports Intel GPU devices.
+
+The following algorithms are implemented:
```

Comment:
Add (probably prepend) something about the ``esimd`` namespace, e.g. "These templates are available in the ``oneapi::dpl::experimental::kt::esimd`` namespace."

Reply:
Done